src/main/java/io/rebloom/client/cms/CMS.java

```diff
@@ -0,0 +1,96 @@
+package io.rebloom.client.cms;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Interface for RedisBloom Count-Min Sketch Commands
+ * 
+ * @see <a href=
+ *      "https://oss.redislabs.com/redisbloom/CountMinSketch_Commands/">RedisBloom
+ *      Count-Min Sketch Documentation</a>
+ */
+public interface CMS {
+	/**
+	 * CMS.INITBYDIM Initializes a Count-Min Sketch to dimensions specified by user.
+	 * 
+	 * @param key   The name of the sketch
+	 * @param width Number of counter in each array. Reduces the error size
+	 * @param depth Number of counter-arrays. Reduces the probability for an error
+	 *              of a certain size (percentage of total count
+	 */
+	void cmsInitByDim(String key, Long width, Long depth);
+
+	/**
+	 * CMS.INITBYPROB Initializes a Count-Min Sketch to accommodate requested
+	 * capacity.
+	 * 
+	 * @param key         The name of the sketch.
+	 * @param error       Estimate size of error. The error is a percent of total
+	 *                    counted items. This effects the width of the sketch.
+	 * @param probability The desired probability for inflated count. This should be
+	 *                    a decimal value between 0 and 1. This effects the depth of
+	 *                    the sketch. For example, for a desired false positive rate
+	 *                    of 0.1% (1 in 1000), error_rate should be set to 0.001.
+	 *                    The closer this number is to zero, the greater the memory
+	 *                    consumption per item and the more CPU usage per operation.
+	 */
+	void cmsInitByProb(String key, Double error, Double probability);
+
+	/**
+	 * CMS.INCRBY Increases the count of item by increment
+	 * 
+	 * @param key       The name of the sketch
+	 * @param item      The item which counter to be increased
+	 * @param increment Counter to be increased by this integer
+	 * @return Count for the item after increment
+	 */
+	Long cmsIncrBy(String key, String item, Long increment);
```

Comment:
If we choose to use varag in `List<Long> cmsIncrBy(String key, Map<String, Long> itemIncrements)`, this will not be necessary.

Reply:
The only issue I have, and I had it with using a Map too, is that there is no literal Map creation in Java so we are forcing the user to do a `new Map...` to pass a single value. I don't know if passing a single item/increment is the most common case but I suspect it might be, in which case I would like to have this method. I was almost tempted to bring in Java Tuples as a dependency or create our data object with a builder interface so a user could do something like:
```
List<Long> result = client.cmsIncrBy("foo", IncrementBuilder.with("a", 2).and("b", 3).build());
```